app/controllers/unsubscribe_controller.rb

```diff
@@ -0,0 +1,16 @@
+class UnsubscribeController < ApplicationController
+  def unsubscribe
+    Unsubscribe.subscription!(subscription)
+    render plain: "deleted #{uuid}", status: 200
```

Comment:
Can we return JSON here? Since we're probably doing that in other places.

I'm also wondering if we should return a status of `204`.

Reply:
I've updated to a `204` as that seems more appropriate 👍